### ISSUE TITLE
Add website footer

### ISSUE THREAD

**Issue by mrbrianevans** [mrbrianevans/social-media-export-analyser#12]
Add a footer to the website and
- include credits to digital ocean for hosting it for free.  "hosted by digital ocean" 
- link to source code repository
- link to contact me/ personal website
- short explaination of what the website is/does

**Comment by mrbrianevans** [comment]
Add the version number of the website. Digital Ocean provides an environment variable at build time called `${COMMIT_HASH}` which gives the latest git commit hash. This should be displayed so that its clear which version of the repository is deployed. This is especially helpful when there have been 1 or more failed deployments. Just a link to the commit which is currently deployed would be helpful.

**Comment by mrbrianevans** [comment]
![image](https://user-images.githubusercontent.com/53117772/154127303-c47ec613-aef4-49a4-87d1-80c8be5ec757.png)

Footer has been added with the git version of the deployment, digital ocean badge, and link to my personal website if anyone wants to contact me.

I struggled to get the layout of the footer on the page within the `<vaadin-app-layout>`. Its not perfect, but at least it shows at the bottom of the content. Could definitely be improved in future.
